src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -28972,29 +28989,29 @@ static void G(string s)
 
             var comp = CreateCompilation(
                 new[] { source },
-                parseOptions: TestOptions.Regular7);
+                parseOptions: TestOptions.Regular7, skipUsesIsNullable: true);
```

Comment:
>skipUsesIsNullable: true [](start = 52, length = 24)

Is there a bug that annotations will be emitted? If so, please add a `// PROTOTYPE` comment.

Reply:
We're not emitting (since there are error diagnostics). The symbols do contain annotations, but I think that is correct (we always bind the same, regardless of LangVersion).

---
In reply to: [209706681](https://github.com/dotnet/roslyn/pull/29225#discussion_r209706681) [](ancestors = 209706681)